app/products/calls/screens/host_controls/host_controls.tsx

```diff
@@ -63,20 +80,35 @@ export const HostControls = ({
     const stopScreensharePress = useCallback(async () => {
         hostStopScreenshare(currentCall.serverUrl, currentCall.channelId, session.sessionId);
         await dismissBottomSheet();
-    }, [currentCall.serverUrl, currentCall.id, session.sessionId]);
+    }, [currentCall.serverUrl, currentCall.channelId, session.sessionId]);
+
+    const profilePress = useCallback(async () => {
+        dismissBottomSheet();
```

Comment:
this should be awaited as `openUserProfile` opens as a bottom sheet and you could have a race condition here